class UtilityError:

    def __init__(self) -> None:
        pass